src/main/java/org/nzvirtual/api/dto/RouteRequest.java

```diff
@@ -24,7 +32,13 @@ public RouteRequest(Long id, String airline, int flightNumber, String departure,
         this.flightNumber = flightNumber;
         this.departure = departure;
         this.arrival = arrival;
-        this.days = days;
+        this.monday = monday;
+        this.tuesday = tuesday;
+        this.wednesday = wednesday;
+        this.thursday = thursday;
+        this.friday = friday;
+        this.saturday = saturday;
+        this.sunday = sunday;
```

Comment:
I guess the question I shot on discord is whether this could be put into a struct or similar, So that you have one object with all of these as properties.